plugin-packages/media/src/video/video-loader.ts

```
@@ -0,0 +1,28 @@
+        videoAsset.data = loadedVideos[i] as HTMLVideoElement;
+        videoAsset.setInstanceId(videos[i].id);
+        renderer.engine.addInstance(videoAsset);
+      }
+
+      for (let i = 0; i < videos.length; i++) {
+        assetManager.assets[videos[i].id] = loadedVideos[i];
+      }
+    }
+
+  }
+}
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)